nvme.c

```diff
@@ -166,6 +166,10 @@ static int open_dev(char *dev)
 		if (!n) {
 			return -1;
 		}
+
+		if (global_device)
```

Comment:
Could this be assigned with a rc global device? Should we return an error for a second global device open without a close?